rapids-cmake/cpm/nvcomp.cmake

```
@@ -111,6 +111,10 @@ function(rapids_cpm_nvcomp)
     rapids_cpm_get_proprietary_binary_url(nvcomp ${version} nvcomp_url)
     if(nvcomp_url)
       rapids_cpm_download_proprietary_binary(nvcomp ${nvcomp_url})
+
+      # unset CPM_DOWNLOAD_ALL if we have a proprietary binary enabled this ensure we actually use
```

This is two separate sentences, right?
```suggestion
      # Unset CPM_DOWNLOAD_ALL if we have a proprietary binary enabled. This ensures we actually use
```